pkgs/applications/office/docear/default.nix

```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation {
   };
 
   nativeBuildInputs = [ makeWrapper ];
-  buildInputs = [ oraclejre ];
+  buildInputs = [ jre8 ];
```

Comment:
https://github.com/NixOS/nixpkgs/pull/61817#discussion_r287624309 claims that this actually needs the Oracle JDK. Though if this is about `sun.*` packages, I *think* OpenJDK 8 would ship those? cc @unode – would you be able to test `docear` with this PR applied?

Reply:
I could sometime next week but have to say that I haven't used docear in 4 years.

Reply:
Thanks, no worries. I take it you are not interested in maintaining the package, then? As it has been unmaintained upstream since 2017 (and upstream [says as much](https://isg.beel.org/blog/2021/05/10/2021-update-for-docears-google-scholar-parser-library-to-fetch-metadata-for-pdf-files/)) and it doesn’t have a maintainer invested in it, perhaps we should simply drop this package instead.

Reply:
I would be fine with dropping. We can always bring it back if someone wants it.

Reply:
Dropped as of the latest push, thank you for adding it in the first place ❤️